saleor/graphql/account/tests/mutations/test_token_create.py

```diff
@@ -221,3 +224,34 @@ def test_create_token_active_user_logged_before(api_client, customer_user, setti
     assert payload["type"] == JWT_REFRESH_TYPE
     assert payload["token"] == customer_user.jwt_token_key
     assert payload["iss"] == build_absolute_uri(reverse("api"))
+
+
+@pytest.fixture
+def users_with_similar_emails():
+    users = [
+        User.objects.create_user("andrew@example.com", "password"),
+        User.objects.create_user("Andrew@example.com", "password"),
+        User.objects.create_user("andrew2@example.com", "password"),
+    ]
+    return users
+
+
+@freeze_time("2020-03-18 12:00:00")
+@pytest.mark.parametrize(
+    "email,expected_user",
+    [
+        ("andrew@example.com", 0),
```

Comment:
Maybe I am wrong but I think, that the implemented logic works as previously:
You provide `Andrew@example.com` and expect that `Andrew@example.com` will be fetched from DB.
As fair as I understand the concept, you should provide `Andrew@example.com` and we should try to find a match, but if we can't then we should try to find `andrew@example.com`
Also, I think that, the registration mutation should be changed to also check&save emails in lower letters.
@timuric Can you confirm that I am talking with sense? 😅

Reply:
Yep, registration should be in lower case, and we need to provide also logic for password reset

Reply:
So it means, that we don't need to handle exception, mentioned in issue, right?

> With exception if the system has two accounts under [test@test.com](mailto:test@test.com) and [Test@test.com](mailto:Test@test.com) then login should happen to the corresponding account provided.